app/javascript/components/editor/Header.tsx

```diff
@@ -25,13 +26,11 @@ Header.Title = ({
   <div className="title">
     <div className="track">{trackTitle}</div>
     <div className="divider">/</div>
-    <div className="exeercise">{exerciseTitle}</div>
+    <div className="exercise">{exerciseTitle}</div>
   </div>
 )
 
-Header.ActionHints = () => (
-  <button className="btn-small hints-btn">Hints</button>
-)
+Header.ActionHints = Hints
```

Comment:
As the hints display is now more involved, I've extracted it to a separate component